dapp/src/components/ConnectWalletModal/ConnectWalletAlert.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react"
+import { Box, Link, VStack } from "@chakra-ui/react"
+import { AnimatePresence, Variants, motion } from "framer-motion"
+import { EXTERNAL_HREF } from "#/constants"
+import {
+  Alert,
+  AlertDescription,
+  AlertTitle,
+  AlertIcon,
+  AlertProps,
+} from "../shared/Alert"
+
+export enum ConnectionAlert {
+  Rejected = "REJECTED",
+  NotSupported = "NOT_SUPPORTED",
+  NetworkMismatch = "NETWORK_MISMATCH",
+  InvalidSIWWSignature = "INVALID_SIWW_SIGNATURE",
+  Default = "DEFAULT",
+}
+
+const CONNECTION_ALERTS = {
+  [ConnectionAlert.Rejected]: {
+    title: "Wallet connection rejected.",
+    description: "If you encountered an error, please try again.",
+  },
+  [ConnectionAlert.NotSupported]: {
+    title: "Not supported.",
+    description:
+      "Only Native SegWit, Nested SegWit or Legacy addresses supported at this time. Please try a different address or another wallet.",
+  },
+  [ConnectionAlert.NetworkMismatch]: {
+    title: "Error!",
+    description:
+      "Incorrect network detected in your wallet. Please choose proper network and try again.",
+  },
+  [ConnectionAlert.Default]: {
+    title: "Wallet connection error. Please try again.",
+    description: (
+      <Box as="span">
+        If the problem persists, contact{" "}
+        <Link
+          // TODO: Define in the new color palette
+          color="#0E61FE"
+          textDecoration="underline"
+          href={EXTERNAL_HREF.DISCORD}
+          isExternal
+        >
+          Acre support
+        </Link>
```

Comment:
Let's add a dot at the end of the sentence.
```suggestion
        </Link>.
```